diff --git a/node_modules/@pkgr/utils/lib/browser.js b/node_modules/@pkgr/utils/lib/browser.js
index b36ffa9..0140c1a 100644
--- a/node_modules/@pkgr/utils/lib/browser.js
+++ b/node_modules/@pkgr/utils/lib/browser.js
@@ -1,7 +1,7 @@
 import { __awaiter } from "tslib";
-import { execSync } from 'node:child_process';
-import path from 'node:path';
-import { fileURLToPath } from 'node:url';
+import { execSync } from 'child_process';
+import path from 'path';
+import { fileURLToPath } from 'url';
 import spawn from 'cross-spawn';
 import picocolors from 'picocolors';
 const OSX_CHROME = 'google chrome';
diff --git a/node_modules/@pkgr/utils/lib/constants.js b/node_modules/@pkgr/utils/lib/constants.js
index a09c9b0..ea782f7 100644
--- a/node_modules/@pkgr/utils/lib/constants.js
+++ b/node_modules/@pkgr/utils/lib/constants.js
@@ -1,5 +1,5 @@
 var _a;
-import { createRequire } from 'node:module';
+import { createRequire } from 'module';
 export const DEV = 'development';
 export const PROD = 'production';
 export const NODE_ENV = (_a = process.env.NODE_ENV) !== null && _a !== void 0 ? _a : DEV;
diff --git a/node_modules/@pkgr/utils/lib/helpers.js b/node_modules/@pkgr/utils/lib/helpers.js
index 3caead9..de43f7f 100644
--- a/node_modules/@pkgr/utils/lib/helpers.js
+++ b/node_modules/@pkgr/utils/lib/helpers.js
@@ -1,5 +1,5 @@
-import fs from 'node:fs';
-import path from 'node:path';
+import fs from 'fs';
+import path from 'path';
 import isGlob from 'is-glob';
 import { CWD, EXTENSIONS, cjsRequire, SCRIPT_RUNNERS, SCRIPT_EXECUTORS, } from './constants.js';
 export const tryPkg = (pkg) => {
diff --git a/node_modules/@pkgr/utils/lib/index.cjs b/node_modules/@pkgr/utils/lib/index.cjs
index 88c446f..0653a4a 100644
--- a/node_modules/@pkgr/utils/lib/index.cjs
+++ b/node_modules/@pkgr/utils/lib/index.cjs
@@ -1,12 +1,12 @@
 'use strict';
 
-var node_child_process = require('node:child_process');
-var path = require('node:path');
-var node_url = require('node:url');
+var node_child_process = require('child_process');
+var path = require('path');
+var node_url = require('url');
 var spawn = require('cross-spawn');
 var picocolors = require('picocolors');
-var node_module = require('node:module');
-var fs = require('node:fs');
+var node_module = require('module');
+var fs = require('fs');
 var isGlob = require('is-glob');
 
 var __async = (__this, __arguments, generator) => {
diff --git a/node_modules/@pkgr/utils/lib/monorepo.js b/node_modules/@pkgr/utils/lib/monorepo.js
index 7126296..7166d1f 100644
--- a/node_modules/@pkgr/utils/lib/monorepo.js
+++ b/node_modules/@pkgr/utils/lib/monorepo.js
@@ -1,5 +1,5 @@
 var _a, _b, _c, _d;
-import path from 'node:path';
+import path from 'path';
 import { tryGlob, tryRequirePkg } from './helpers.js';
 const pkg = (_a = tryRequirePkg(path.resolve('package.json'))) !== null && _a !== void 0 ? _a : {};
 const lernaConfig = (_b = tryRequirePkg(path.resolve('lerna.json'))) !== null && _b !== void 0 ? _b : {};
