diff --git a/node_modules/@pkgr/rollup/lib/cli.js b/node_modules/@pkgr/rollup/lib/cli.js
index 6b87f60..3c7047e 100755
--- a/node_modules/@pkgr/rollup/lib/cli.js
+++ b/node_modules/@pkgr/rollup/lib/cli.js
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
-import path from 'node:path';
-import { fileURLToPath } from 'node:url';
+import path from 'path';
+import { fileURLToPath } from 'url';
 import { tryRequirePkg } from '@pkgr/utils';
 import { program } from 'commander';
 import debug from 'debug';
diff --git a/node_modules/@pkgr/rollup/lib/config.js b/node_modules/@pkgr/rollup/lib/config.js
index 8a60fcc..8e22ac4 100644
--- a/node_modules/@pkgr/rollup/lib/config.js
+++ b/node_modules/@pkgr/rollup/lib/config.js
@@ -1,5 +1,5 @@
-import fs from 'node:fs';
-import path from 'node:path';
+import fs from 'fs';
+import path from 'path';
 import { entries } from '@pkgr/es-modules';
 import { getGlobals, normalizePkg, upperCamelCase, } from '@pkgr/umd-globals';
 import { CWD, EXTENSIONS, PROD, __DEV__, __PROD__, arrayify, identify, monorepoPkgs, tryExtensions, tryFile, tryGlob, tryPkg, tryRequirePkg, } from '@pkgr/utils';
