diff --git a/node_modules/ts-jest/dist/transpilers/typescript/transpile-module.js b/node_modules/ts-jest/dist/transpilers/typescript/transpile-module.js
index e539389..6dbfb54 100644
--- a/node_modules/ts-jest/dist/transpilers/typescript/transpile-module.js
+++ b/node_modules/ts-jest/dist/transpilers/typescript/transpile-module.js
@@ -54,9 +54,9 @@ const shouldCheckProjectPkgJsonContent = (fileName, moduleKind) => {
  * - Allow using custom AST transformers with the internal created {@link Program}
  */
 const transpileWorker = (input, transpileOptions) => {
-    barebonesLibSourceFile ??= typescript_1.default.createSourceFile(barebonesLibName, barebonesLibContent, {
+    ;(barebonesLibSourceFile == null ? (barebonesLibSourceFile = typescript_1.default.createSourceFile(barebonesLibName, barebonesLibContent, {
         languageVersion: typescript_1.default.ScriptTarget.Latest,
-    });
+    })): barebonesLibSourceFile);
     const diagnostics = [];
     const options = transpileOptions.compilerOptions
         ? // @ts-expect-error internal TypeScript API
